frontend/web/components/modals/CreateFlag.js

```diff
@@ -84,7 +84,7 @@ const CreateFlag = class extends Component {
       name,
       period: 30,
       selectedIdentity: null,
-      tab: Utils.fromParam().tab || 0,
```

Comment:
Are there any repercussions to changing this? I don't know how `Utils.fromParam().tab` was being set / used before.

Reply:
There is no problem because it was not being used before either.